php/Admin/Upgrade.php

```diff
@@ -52,10 +52,8 @@ public function enqueue_scripts() {
 			return;
 		}
 
-		$page = filter_input( INPUT_GET, 'page' );
-
 		// Enqueue scripts and styles on the edit screen of the Block post type.
-		if ( $this->slug === $page ) {
+		if ( filter_input( INPUT_GET, 'page' ) === $this->slug ) {
```

Comment:
Because this isn't sanitized, there shouldn't be a variable `$page` that can be used for anything. 

`filter_input()` should just run inline.